superset-frontend/src/dashboard/components/DashboardBuilder.jsx

```diff
@@ -64,7 +64,6 @@ const propTypes = {
 };
 
 const defaultProps = {
-  showBuilderPane: false,
```

Comment:
seems less risky to remove the `.isRequired` here too  (unless you can confirm you checked all the calls)

Reply:
I did, but you're right, let's keep the default and remove isRequired

Reply:
I was confused about the type here and tracked it into the child component, and it looks unused there. Can you double check and remove all references to it here if that's the case?

Reply:
It seems that not only `showBuilderPane` was unused, but also `colorScheme` and `setColorSchemeAndUnsavedChanges`. I removed references from this file, containers/DashboardBuilder and the spec file.